### DISCUSSION TITLE
Cannot get out of Game

### DISCUSSION THREAD

**Discussion by hendrikbez** [dougmencken/HeadOverHeels#63] [General]
Thank you, did install it now in my virtual box with ubuntu.
I can run the game, but I can not exit or do save in it, does not go back to main screen. I do not see any keys to go back.  What am I doing wrong

**Comment by dougmencken** [comment]
Once the game begins, you can’t quit wahaha 😁

But really, I think the “pause” key is what you’re l👀kin’ for. By default it’s [ m ]† but can be redefined

![keys.png](https://github.com/dougmencken/HeadOverHeels/assets/3457110/d85018e9-848b-410b-93f3-c3b1163806d0)

† _And if you’re wondering why I designated (M) for pause, you’ve never seen [a ZX Spectrum keyboard](https://retrorepairsandrefurbs.files.wordpress.com/2021/07/img_6053.jpg)_ 😏
![m for pause](https://github.com/dougmencken/HeadOverHeels/assets/3457110/d71be740-19f6-4230-846e-6b5732736648)

When the game is paused, the “escape” [ esc ] key finishes the game and brings the player back to the menus

![pause.png](https://github.com/dougmencken/HeadOverHeels/assets/3457110/e8857779-4fe6-404e-a80d-7ab3d0cd9d24)

**Comment by dougmencken** [comment]
As for saving the in-game progress, you either eat the reincarnation fish found in some game rooms exactly like in the original Ritman and Drummond’s game, either cheat with [ alt ] & [ shift ] & [ s ]

![saving.png](https://github.com/dougmencken/HeadOverHeels/assets/3457110/3dc09787-c05e-4ba7-9460-23d848958dc4)

For some testing-only (and no other ;) purposes, the initial rooms can be set as arguments to the game’s binary
```
--head-room=(value) — the room's file name where Head begins the game, =blacktooth1head.xml by default
--heels-room=(value) — the room's file name where Heels begins the game, =blacktooth23heels.xml by default
```
implying that you know which room is described by which file [somewhere deep in ``gamedata/map``](https://github.com/dougmencken/HeadOverHeels/tree/master/gamedata/map)

**Reply by dougmencken** [reply]
and since https://github.com/dougmencken/HeadOverHeels/commit/0cd004f0b43e9fa47d6f820dfa4373cd86bf954a the line “save the game” is animated in cycling colors (currently english-only, ah)

![cyclingcolor](https://github.com/dougmencken/HeadOverHeels/assets/3457110/81fa9b40-113f-4af6-8eb1-66667ef7e4ff)

**Comment by dougmencken** [comment]
With the new wiki page https://github.com/dougmencken/HeadOverHeels/wiki/Questions-&-Answers I’m closing this #63
